.github/workflows/ci.yml

```
@@ -62,22 +62,61 @@ jobs:
+            
       - name: Build APK (Android)
-        run: flutter build apk
-
+        if: ${{ matrix.platform == 'ubuntu-latest' }}
+        run: |
+          flutter build apk --release \
+            --dart-define=FB_APP_ID=${{ secrets.FB_APP_ID }} \
+            --dart-define=FB_APP_NAME=${{ secrets.FB_APP_NAME }} \
+            --dart-define=GITHUB_OAUTH_CLIENT_ID=${{ secrets.GH_OAUTH_CLIENT_ID }} \
+            --dart-define=GITHUB_OAUTH_CLIENT_SECRET=${{ secrets.GH_OAUTH_CLIENT_SECRET }}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)